src/js/index.js

```diff
@@ -186,6 +186,9 @@ export const hpe = deepFreeze({
       border: {
         radius: '4px',
       },
+      extend: ({ theme }) =>
+        `margin-top: ${theme.global.edgeSize.xsmall}; 
+       margin-bottom: ${theme.global.edgeSize.xsmall};`,
```

Comment:
Just a formatting suggestion.

```suggestion
      extend: ({ theme }) => `
        margin-top: ${theme.global.edgeSize.xsmall};
        margin-bottom: ${theme.global.edgeSize.xsmall};
      `,
```